queries/scala/function_arguments.scm

```diff
@@ -0,0 +1,8 @@
+(function_definition
+  name: (identifier) @function)
+
+(parameter
+  name: (identifier) @parameter)
+
+(binding
+  name: (identifier) @parameter)
```

Comment:
```suggestion
(parameter
  name: (identifier) @argname)

(binding
  name: (identifier) @argname)
```

No need to query for the function definition here. Also I changed the capture name to `argname` for consistency (some capture names are used for internal logic)